config/file_config.go

```diff
@@ -185,12 +185,13 @@ type RefineryTelemetryConfig struct {
 }
 
 type TracesConfig struct {
-	SendDelay    Duration `yaml:"SendDelay" default:"2s"`
-	BatchTimeout Duration `yaml:"BatchTimeout" default:"100ms"`
-	TraceTimeout Duration `yaml:"TraceTimeout" default:"60s"`
-	MaxBatchSize uint     `yaml:"MaxBatchSize" default:"500"`
-	SendTicker   Duration `yaml:"SendTicker" default:"100ms"`
-	SpanLimit    uint     `yaml:"SpanLimit"`
+	SendDelay        Duration `yaml:"SendDelay" default:"2s"`
+	BatchTimeout     Duration `yaml:"BatchTimeout" default:"100ms"`
+	TraceTimeout     Duration `yaml:"TraceTimeout" default:"60s"`
+	MaxBatchSize     uint     `yaml:"MaxBatchSize" default:"500"`
+	SendTicker       Duration `yaml:"SendTicker" default:"100ms"`
+	SpanLimit        uint     `yaml:"SpanLimit"`
+	MaxExpiredTraces uint     `yaml:"MaxExpiredTraces" default:"5000"`
```

Comment:
I wonder if the default should be 0 which is no limit?

Reply:
I thought about that, but since we know this is causing problems and that 5000 with 100ms SendTicker caps the loop around 3s and that our default Healthcheck ticker is 3s, I figured this will be useful for most users instead of them needing to configure it themselves.